Sources/JavaKitMacros/JavaClassMacro.swift

```diff
@@ -87,13 +87,21 @@ extension JavaClassMacro: MemberMacro {
       }
       """
 
+    let nonOptionalAs: DeclSyntax = """
+      /// Casting to <\(raw: superclass)> will never be nil because <\(raw: className.split(separator: ".").last!)> extends it.
+      public func `as`(_: \(raw: superclass)) -> \(raw: superclass) {
```

Comment:
With the `as` API, one always passes a metatype, e.g,

```swift
obj.as(MyJavaClass.self)
```

So I think the type of the parameter to `as` needs to be `\(raw: superclass).Type` .